### DISCUSSION TITLE
Add image or color to background

### DISCUSSION THREAD

**Discussion by rafaelleonan** [oguzhnatly/flutter_carplay#44] [Q&A] [answered]
Would it be possible to add a background image to the app within CarPlay or just change the color?
Just like the example below:

<img width="872" alt="Captura de Tela 2023-09-18 às 16 15 43" src="https://github.com/oguzhnatly/flutter_carplay/assets/55757244/54320efb-1c17-4157-815c-5bdb44a90f55">

**Comment by redFrenesi** [comment]
hi you cant do it, because apple dont want it

**Comment by oguzhnatly** [comment, marked as answer]
CarPlay's UI is controlled by Apple's design system and doesn't support custom background images or colors on templates. This is an Apple restriction, not a plugin limitation. CarPlay enforces a consistent look across all apps for driver safety.

The only customization available is through the template content itself (list items, grid buttons, images within those components).
